spring-batch-2/pom.xml

```
@@ -104,6 +104,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>com.graphql-java</groupId>
+            <artifactId>java-dataloader</artifactId>
+            <version>3.2.0</version>
```

This can be done asynchronously, but let's pull this out to be a property, like the other dependencies.


```suggestion
            <version>${java-dataloader.version}</version>
```